doc/md/tutorial/helm-values_test.go

```diff
@@ -22,10 +22,10 @@ func TestMain(m *testing.M) {
 }
 
 // Run these with go test -v to see the verbose names
-func TestMarkdown(t *testing.T) {
-	t.Run("AddOnPromoter", func(t *testing.T) {
+func TestHelmValues(t *testing.T) {
+	t.Run("TestHelmValues", func(t *testing.T) {
```

Comment:
Latent copypasta from the AddOnPromoter improvements that should have been changed previously